instrumentation/jdbc/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jdbc/JdbcNetAttributesExtractor.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jdbc;
+
+import io.opentelemetry.instrumentation.api.instrumenter.InetSocketAddressNetAttributesExtractor;
+import java.net.InetSocketAddress;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+final class JdbcNetAttributesExtractor
+    extends InetSocketAddressNetAttributesExtractor<DbRequest, Void> {
+  @Override
+  protected @Nullable InetSocketAddress getAddress(DbRequest dbRequest, Void unused) {
+    String host = dbRequest.getDbInfo().getHost();
+    Integer port = dbRequest.getDbInfo().getPort();
+    if (host != null && port != null) {
+      return new InetSocketAddress(host, port);
```

Comment:
We should either directly implement `NetAttributesExtractor` or at least create unresolved, we don't want to do DNS lookup to populate an attribute